tests/test_admin.py

```diff
@@ -88,6 +88,19 @@ def test_change_form_fields(self):
         self.assertIn('name="expires_0"', decoded_response)
         self.assertIn('name="expires_1"', decoded_response)
 
+    def test_change_form_title(self):
+        """
+        Ensure that the change form title is as expected
```

Comment:
Again pedantic (sorry), but this could be reworded as 

```suggestion
        The change form title is populated with the custom value
```